ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -47,5 +49,34 @@ final class Metrics(override val registry: MetricRegistry, val meter: OtelMeter)
 
     object HttpJsonApi extends HttpJsonApiMetrics(prefix :+ "http_json_api", registry)
 
+    object grpc extends OpenTelemetryFactory with GrpcServerMetrics {
```

Comment:
Not new code from this PR, but I don't think the `daml` object should be a `DropwizardFactory`.
Making it a simple object would remove the strange fact of having an `OpenTelemetryFactory` inside a `DropwizardFactory`.

Reply:
I fully agree. I wouldn't touch this in this PR though as the follow up PR that I have for integrating the metrics do in fact refactor this code.